code/src/App.js

```diff
@@ -1,9 +1,21 @@
 import React from 'react';
+import { Provider } from 'react-redux';
+import { configureStore, combineReducers } from '@reduxjs/toolkit';
+import AddToDo from 'components/AddToDo';
+import ToDoList from 'components/ToDoList';
+import todos from 'redux/reducers/todos'
 
 export const App = () => {
+  const reducer = combineReducers({
+    todos: todos.reducer
+  });
+  const store = configureStore({ reducer });
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  );
+    <Provider store={store}>
+      <AddToDo />
+      <ToDoList />
+    </Provider>
+  )
 }
+
+// ctrl + c inside the terminal to stop the liveServer
```

Comment:
Nice that you write such comments but I think our final code should be without comments, however, I strongly advise you to make a separate google sheet - docs - where you put all your coding notes ;)